internal/filegreeter/greeter_test.go

```diff
@@ -0,0 +1,41 @@
+package filegreeter
+
+import (
+	"bytes"
+	"context"
+	"testing"
+)
+
+func TestFileGreeter_Greet(t *testing.T) {
```

Comment:
i see you discovered the test generate tool in goland ;)

Reply:
haha, yea its pretty nice